pages/curriculum.tsx

```diff
@@ -50,6 +50,7 @@ const Curriculum: React.FC = () => {
           challengeCount={e.challenges.length}
           hourCount="-"
           description={e.description}
+          currentState="completed"
         ></LessonCard>
```

Comment:
`<LessonCard />` [if you don't have children please do one self closing tag ](https://reactjs.org/docs/jsx-in-depth.html)